hudi-common/src/main/java/org/apache/hudi/common/engine/HoodieReaderContext.java

```diff
@@ -141,6 +141,7 @@ public void setShouldMergeUseRecordPosition(boolean shouldMergeUseRecordPosition
   public static final String INTERNAL_META_OPERATION = "_3";
   public static final String INTERNAL_META_INSTANT_TIME = "_4";
   public static final String INTERNAL_META_SCHEMA = "_5";
+  public static final String PROCESSING_TIME_BASED_DELETE_FOUND = "_6";
```

Comment:
lets call this DELETE_FOUND_WITHOUT_ORDERING_VALUE 
so its more clearer

Reply:
Done